runtime_lib/airhost/host.cpp

```diff
@@ -325,6 +325,13 @@ uint64_t air_segment_load(const char *name) {
 
 #ifdef AIR_PCIE
   XAie_Finish(&(_air_host_active_libxaie->DevInst));
+
+  // Setting the driver libxaie backend back up
+  // Currently only targetting device 0
+  _air_host_active_libxaie->AieConfigPtr.Backend = XAIE_IO_BACKEND_AMDAIR;
+  _air_host_active_libxaie->DevInst.IOInst =
+      (void *)"/sys/class/amdair/amdair/00";
```

Comment:
I'm not really familiar with this function - what needs to change to make it work with multiple devices?

Reply:
Segments are a higher-level construct, and segment_load load is called within AIR-level tests. To support multiple devices, we would need the runtime to choose a device where it could load the segment. I have some ideas, lets discuss offline.